mifosng-android/src/main/res/values/styles.xml

```diff
@@ -10,7 +10,7 @@
         <!-- Customize your theme here. -->
     </style>
 
-    <style name="MaterialAppTheme" parent="Theme.AppCompat.DayNight.NoActionBar">
+    <style name="MaterialAppTheme" parent="Theme.MaterialComponents.DayNight.NoActionBar">
```

Comment:
have you tested this on multiple android versions?

Reply:
No,
how to check that?
by downloading different version android on android studio?

Reply:
create virtual devices with multiple dpis. [Checkout this](https://developer.android.com/studio/run/managing-avds)